pkg/administration/events/hints.go

```diff
@@ -17,7 +17,8 @@ var (
 	hints = map[string]map[string]map[string]string{
 		AuthenticationDomain: {
 			adminResources.APIToken: {
-				"": `An API token is about to expire. See the details on the expiration time within the event message.
+				codes.TokenCreated: `An API token has been created.`,
+				codes.TokenExpired: `An API token is about to expire. See the details on the expiration time within the event message.
```

Comment:
Has the change of trigger for expired token been (e2e) tested ?
I think the expiration date of a token can now be selected at creation time.

Reply:
I don't think so. But there is a new test for the token creation, which executes a similar path.